src/test/resources/data/AutomatedSessionRemindersTest.json

```diff
@@ -22,7 +22,7 @@
       "googleId": "AutSessRem.instructor",
       "courseId": "AutSessRem.course",
       "name": "Test Ins for Aut Sessions Reminder",
-      "email": "${support.email}",
+      "email": "${test.instructor}@gmail.com",
```

Comment:
Yes I know about that, it was documented in `AutomatedSessionRemindersTest.java`, and I also did the testing on staging server myself. However the point here is why _specifically_ support email address while other email accounts work just fine (I have also tested this).